Extensions/xneko.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE XNeko **//
-//* VERSION 1.2 REV A **//
+//* VERSION 1.2.2 **//
 //* DESCRIPTION A cat for your dashboard! **//
 //* DEVELOPER STUDIOXENIX **//
```

Comment:
This should be changed to `//* DEVELOPER new-xkit **//` for our organization. :)